modules/quality_control/readme.md

```diff
@@ -0,0 +1,28 @@
+## Imaging Quality Control Module
+
+#### Purpose
+The purpose of this module is to verify and help ensure imaging data has been uploaded into LORIS and reviwed for Quality Control (QC). It provides a queryable interface for users to assess the completeness of imaging data collections matched against a given protocol, and the status of QC review processes on them.
```

Comment:
@christinerogers is this line necessary?

Reply:
typo: `reviwed`  
instead of removing it, let's make it better: `matched against a given protocol` -> `for pre-defined scan types`.  yes, i think it's important to keep the idea that completeness is measured against the protocol / mri parameter form

Reply:
^ @knoll-alex these were my inline comments, you should still be able to see them. I fixed them anyway just now.

Reply:
the most recent commits should resolve this.  waiting on re-review.